main/src/ca/uwaterloo/flix/api/lsp/provider/CodeActionProvider.scala

```diff
@@ -53,8 +53,8 @@ object CodeActionProvider {
     case ResolutionError.UndefinedTrait(qn, ap,  _, loc) if overlaps(range, loc) =>
       mkUseTrait(qn.ident, uri, ap)
 
-    case ResolutionError.UndefinedTag(name, _, loc) if overlaps(range, loc) =>
-      replaceFullTag(name, uri, loc)
+    case ResolutionError.UndefinedTag(name, ap, _, loc) if overlaps(range, loc) =>
+      mkUseTag(name, uri, ap) ++ replaceFullTag(name, uri, loc)
```

Comment:
replaceFullTag -> mkQualifyTag ?

Reply:
Also lets change "Replace" in the message to "Prefix with"

Reply:
(I know these two comments applied more to the old PR, but I just saw these things now.)